sendgrid/helpers/mail/mail.py

```diff
@@ -376,9 +376,9 @@ def categories(self):
         return self._categories
 
     def add_category(self, category):
-        """Add a Category to this Mail.  Must be less than 255 characters.
```

Comment:
What was your reason behind this deletion?

Reply:
Since Category is a Category type `Must be less than 255 characters.` is not suitable.